README.md

```diff
@@ -1,41 +1,158 @@
+- [Release process](#release-process)
+- [Release history](#release-history)
+- [License](#license)
+- [Research paper](#paper)
+- [Contributing](#contributing)
 
-1. Download the [Habitat pre-trained
-   models](https://doi.org/10.5281/zenodo.4876277).
-2. Run `extract-models.sh` under `analyzer` to extract and install the
-   pre-trained models.
-3. Run `setup.sh` under `docker/` to build the Habitat container image.
-4. Run `start.sh` to start a new container. By default, your home directory
-   will be mounted inside the container under `~/home`.
-5. Once inside the container, run `install-dev.sh` under `analyzer/` to build
-   and install the Habitat package.
-6. In your scripts, `import habitat` to get access to Habitat. See
-   `experiments/run_experiment.py` for an example showing how to use Habitat.
+Habitat is a tool that predicts a deep neural network's training iteration execution time on a given GPU. It currently supports PyTorch. To learn more about how Habitat works, please see our [research paper](https://arxiv.org/abs/2102.00527).
 
-**Note:** Habitat needs access to your GPU's performance counters, which
-requires special permissions if you are running with a recent driver (418.43 or
-later). If you encounter a `CUPTI_ERROR_INSUFFICIENT_PRIVILEGES` error when
-running Habitat, please follow the instructions
-[here](https://developer.nvidia.com/ERR_NVGPUCTRPERM)
-and in [issue #5](https://github.com/geoffxy/habitat/issues/5).
+<h2 id="installation">Installation</h2>
 
+To run Habitat, you need:
+- [Python 3.6+](https://www.python.org/)
+- [Pytorch 1.1.0+](https://pytorch.org/)
+- A system equiped with an Nvidia GPU.
 
-## License
+Currently, we have predictors for the following Nvidia GPUs:
+
+| GPU        | Generation  | Memory | Mem. Type | SMs |
+| ---------- |:-----------:| ------:| :-------: | :-: |
+| P4000      | Pascal      | 8 GB   | GDDR5     | 14  |
+| P100       | Pascal      | 16 GB  | HBM2      | 56  |
+| V100       | Volta       | 16 GB  | HBM2      | 80  |
+| 2070       | Turing      | 8 GB   | GDDR6     | 36  |
+| 2080Ti     | Turing      | 11 GB  | GDDR6     | 68  |
+| T4         | Turing      | 16 GB  | GDDR6     | 40  |
+| 3090       | Ampere      | 24 GB  | GDDR6X    | 82  |
+
+**NOTE:** Not implmented yet
+```zsh
+python3 -m pip install habitat
+python3 -c "import habitat"
+```
+
+<h2 id="build">Building from source</h2>
+
+Prerequsites:
+- A system equiped with an Nvidia GPU with properly configured CUDA
+- [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit-archive)
+- [cmake v3.17+](https://github.com/Kitware/CMake/releases)
+- [Habitat pre-trained models](https://zenodo.org/record/4876277)
+
+```zsh
+git clone https://github.com/CentML/habitat.git && cd habitat
+git submodule init && git submodule update
+
+# Download the pre-trained models
+cd analyzer
+curl -O https://zenodo.org/record/4876277/files/habitat-models.tar.gz\?download\=1
+
+# Install the models
+./extract-models.sh
```

Comment:
Doesn't need to be addressed in this PR, but something to think about - is it possible to wrap these installation steps inside a `setup.py` so that the user can automate the installation via pip or conda?